ros/api/v0/hosts.py

```diff
@@ -107,10 +107,12 @@ class HostDetailsApi(Resource):
     profile_fields = {
         'host_id': fields.String(attribute='inventory_id'),
         'performance_record': fields.String,
```

Comment:
While testing I found `performance_record` is getting set to null. We can also remove `performance_record` from here.

Reply:
done.